docker-compose.yaml

```diff
@@ -0,0 +1,103 @@
+# Copyright (C) 2018 spdfg
+#
+# This file is part of Elektron.
+#
+# Elektron is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Elektron is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Elektron.  If not, see <http://www.gnu.org/licenses/>.
+#
+version: "2"
+
+services:
+  zk:
+    image: rdelvalle/zookeeper
+    restart: on-failure
+    ports:
+    - "2181:2181"
+    environment:
+      ZK_CONFIG: tickTime=2000,initLimit=10,syncLimit=5,maxClientCnxns=128,forceSync=no,clientPort=2181
+      ZK_ID: 1
+    networks:
+      elektron-cluster:
+        ipv4_address: 19.16.4.3
+    logging:
+      driver: none
+        
+  mesos-master:
+    image: rdelvalle/mesos-master:1.5.1
+    restart: on-failure
+    ports:
+    - "5050:5050"
+    environment:
+      MESOS_ZK: zk://19.16.4.3:2181/mesos
+      MESOS_QUORUM: 1
+      MESOS_HOSTNAME: localhost
+      MESOS_CLUSTER: test-cluster
+      MESOS_REGISTRY: replicated_log
+      MESOS_WORK_DIR: /tmp/mesos
+    networks:
+      elektron-cluster:
+        ipv4_address: 19.16.4.4
+    logging:
+      driver: none
+    depends_on:
+    - zk
+
+  mesos-agent:
+    image: pkaushi1/mesos-agent-elektron:1.5.1
```

Comment:
We own the organization `spdf` at dockerhub https://hub.docker.com/orgs/spdf/repositories

Reply:
Good to know. I'll push the image there then.